src/styles.css

```diff
@@ -0,0 +1,91 @@
+body {
+    margin: 0px;
+}
+
+#worldmap {
+    width: 95%;
+
+
+}
+
+#earth{
+    font-family: "Brush Script MT";
```

Comment:
Hier mist nog de fallback font als de Brush Script MT niet gevonden kan worden, wat helaas bij mij het geval is